src/Autumn/aexpr.jl

```diff
@@ -111,7 +125,20 @@ function showstring(expr::Expr)
   end
 end
 
+showstring(lst::Array{}) = lst
+showstring(str::String) = str
+needequals(val) = "="
+
+function needequals(val)
```

Comment:
These two methods have the same type and hence the latter is overriding the former.

See the build warning https://travis-ci.org/github/zenna/CausalDiscovery.jl/jobs/692012502